chart/templates/secrets/metadata-connection-secret.yaml

```
@@ -19,7 +19,7 @@
 ## Airflow Metadata Secret
 #################################
 {{- if not .Values.data.metadataSecretName }}
-{{- $metadataHost := .Values.data.metadataConnection.host | default (printf "%s-%s.%s" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $metadataHost := .Values.data.metadataConnection.host | default (include "airflow.default.postgres.connection.string" .) }}
```

It might be cleaner to just have this all live here.

```suggestion
{{- $defaultMetadataHost :=  .Values.postgresql.nameOverride | default (printf "%s-%s.%s" .Release.Name "postgresql" .Release.Namespace) }}
{{- $metadataHost := .Values.data.metadataConnection.host | default $defaultMetadataHost }}
```

---

Makes sense. We can have it like that too.

---

Pushed a fix containing this suggestion @jedcunningham